shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -677,6 +718,13 @@ - (void)notifyLowMemory {
 
 #pragma mark - Text input delegate
 
+- (void)handlePressEvent:(FlutterUIPressProxy*)press
```

Comment:
Text input was working, but it wasn't using key events, it was using the IME methods, and consequently events were not being sent when text was entered into a field. This function is here because both the `TextInputPlugin` and the `FlutterViewController` override need to override native methods to receive key events, and this function allows the text input plugin to talk to the view controller so that they can share the implementation of `handlePressEvent`.